README.md

```diff
@@ -1,2 +1,38 @@
 # comp110-worksheet-5
 Base repository for COMP110 worksheet 5
+
+a)
+	Checks the whole list for duplicate.
+	if the i and j position arent the same but the value in the list is than it will return true.
+
+b)
+	For every extra element in the list it will quadruple the running time.
+	The reason being that every element has to be cross checked with each other.
```

Comment:
✔ (although it's doubling the size of the list that will quadruple the running time, not adding one)